### PR TITLE
ch(coveralls): integrate coveralls

### PR DESCRIPTION
**Pull request by klevamane**
#### What does this PR do?
Integrate coveralls code coverage
#### Description of Task to be completed?
- add .coveralls.yml file
- add coveralls badge to readme
#### How should this be manually tested?
- Check the badge in the readme
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#159232379

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: EttJVT8sAgYCCrcPnC5Cb94xsTHG3oQ7C
```
Comment (OKiMaureen) [inline .coveralls.yml]: Good job, However, we can add a newline to the end of the file.

Comment (anuonifade) [inline .coveralls.yml]: @klevamane @OKiMaureen Why is this here?

**Review by anuonifade — CHANGES_REQUESTED** [review]
Please ensure sensitive information is referenced through an environment variable.

File: tests/models/initial.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    it('Should return error 404', function(done) {
+      chai.request(server)
+        .get('/api/bad-route')
+        .end(function(err, res) {
+          expect(res.status).to.equal(404);
+          done();
+        });
+    });
+  });
+});
```
[the remark this answers is not in the document]
Comment (anuonifade) [reply tests/models/initial.test.js]: @klevamane what's up with this? Why is it not fixed?

**Comment by klevamane** [comment]
The Implementation has already been done.

File: tests/models/initial.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (klevamane) [reply tests/models/initial.test.js]: @andela-aonifade I've already fixed it. See the changes below on checks that have passed.

**Review by OKiMaureen — APPROVED** [review]

File: app.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import express from 'express';
```
Comment (veeqtor) [inline app.js]: This app.js file is not needed here, the index.js file is already doing the work.

**Review by veeqtor — APPROVED** [review]

**Review by anuonifade — APPROVED** [review]

**Pull request merged**